ui/src/app/views/admin-user/component.ts

```diff
@@ -50,6 +63,13 @@ export class AdminReviewIdVerificationComponent implements OnInit {
     }
   }
 
+  disableUser(disable: boolean, profile: Profile): void {
+    this.authDomainService.disableUser('registered',
```

Comment:
(Side note) Oof.. it's not great naming for our API method "disableUser" to actually be for disabling *or* enabling the user. This led me astray for a minute, and I thought we needed a separate method for enabling users... until I realized this is just an updateUserDisabledState method in disguise :)

I suppose it would be a pain to rename it at this point, but small mental hurdles like this can add up quickly & make it gummy to work in a codebase. Maybe I'll take a tech debt item at some point and work through some of these.

Reply:
I... think it does make sense to rename it and refactor away group name. Let me do a quick look into it to see if this is actually used anywhere.

Reply:
Follow up in this ticket:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2131&epics=visible